CMakeLists.txt

```diff
@@ -210,21 +210,22 @@ source_group(TREE ${GENERATORS_ROOT} FILES ${generator_srcs})
 source_group("Sources" FILES ${test_srcs})
 source_group("Sources" FILES ${python_srcs})
 
-set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT Tests)
+set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT unit_tests)
 
 enable_testing()
 
-add_executable(
+target_include_directories (
   unit_tests
-  ${TEST_SRCS}
-  ${GENERATOR_SRCS}
+  PRIVATE
+  ${CMAKE_SOURCE_DIR}/src
 )
 
+
 target_link_libraries(
-  onnxruntime-genai
   unit_tests
+  PRIVATE
   GTest::gtest_main
```

Comment:
Here we need to link to GenAI static libraries, and all the dependencies that GenAI uses, such as SPM.

Reply:
Is there a variable for that, or do we need to list them out?

Reply:
No such a thing yet. We can do it later.